aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -21,9 +21,9 @@ template<int depth, typename T>
 __device__ bool init_args(
     T** args,
     TensorListMetadata<depth>& tl,
-    int chunk_idx,
-    int chunk_size,
-    int tensor_loc) {
+    const int64_t chunk_idx,
```

Comment:
what about perf impact of this, any benchmarks?

Reply:
will do